tools/bin/go_core_tests

```diff
@@ -4,22 +4,14 @@ set +e
 
 SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
 OUTPUT_FILE=${OUTPUT_FILE:-"./output.txt"}
+EXTRA_FLAGS = ""
```

Comment:
Logs show it tripping over this, but still running?

Reply:
Remove the spaces surrounding the equal sign